pkgs/tools/misc/togglesg-download/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, buildPythonApplication, makeWrapper, ffmpeg }:
+
+buildPythonApplication rec {
+
+  name = "togglesg-download-git-${version}";
+  version = "2016-02-08";
+
+  src = fetchFromGitHub {
+    owner = "0x776b7364";
+    repo = "toggle.sg-download";
+    rev = "5cac3ec039d67ad29240b2fa850a8db595264e3d";
+    sha256 = "0pqw73aa5b18d5ws4zj6gcmzap6ag526jrylqq80m0yyh9yxw5hs";
+  };
+
+  buildInputs = [ makeWrapper ];
```

Comment:
On Sun, Mar 27, 2016, at 06:11 AM, Peter Hoeg wrote:
> > @@ -0,0 +1,40 @@
> > +{ stdenv, fetchFromGitHub, buildPythonApplication, makeWrapper, ffmpeg }:
> > +
> > +buildPythonApplication rec {
> > +
> > +  name = "togglesg-download-git-${version}";
> > +  version = "2016-02-08";
> > +
> > +  src = fetchFromGitHub {
> > +    owner = "0x776b7364";
> > +    repo = "toggle.sg-download";
> > +    rev = "5cac3ec039d67ad29240b2fa850a8db595264e3d";
> > +    sha256 = "0pqw73aa5b18d5ws4zj6gcmzap6ag526jrylqq80m0yyh9yxw5hs";
> > +  };
> > +
> > +  buildInputs = [ makeWrapper ];
> 
> I thought ```nativeBuildInputs``` were only used when cross compiling.
> What difference would it make here?

Cross compiling isn't really supported, afaik, so it actually never
matters. I do, however, think it is good style to distinguish build-time
only dependencies, which `makeWrapper` certainly is. It's just a
suggestion, feel free to ignore it.